themes/src/main/resources/theme/keycloak-preview/account/resources/app/content/signingin-page/SigningInPage.tsx

```diff
@@ -234,8 +228,19 @@ class SigningInPage extends React.Component<SigningInPageProps, SigningInPageSta
             </React.Fragment>)
     }
 
+    private credentialRowCells(credential: UserCredential, type: string): React.ReactNode[] {
+        const credRowCells: React.ReactNode[] = [];
+        credRowCells.push(<DataListCell id={`${SigningInPage.credElementId(type, credential.id, 'label')}`} key={'userLabel-' + credential.id}>{credential.userLabel}</DataListCell>);
+        if (credential.createdDate) {
```

Comment:
I think if `createDate` is `-1`, this will evaluate to true.
```suggestion
        if (credential.strCreatedDate) {
```

Reply:
Scratch that. `-1` is also a falsey value.

But still, `strCreatedDate` would probably be more appropriate here as we have a more complex condition for that above.

Reply:
Good catch.